package.json

```diff
@@ -652,7 +712,71 @@
         }
       }
     ],
+    "submenus": [
+      {
+          "id": "sequencetype",
+          "label": "New Pass"
+      },
```

Comment:
What is this for?

Reply:
Copied it from vscode-nlp and didn't delete it.